tasks/39_unique-in-order/unique-in-order.js

```diff
@@ -0,0 +1,3 @@
+export function uniqueInOrder(str) {
+  return [].filter.call(str, (el, i) => el != str[i+1]);
```

Comment:
Can this be solved using for loop?

Reply:
Yes, I changed the code to use a `for` loop